pkgs/development/libraries/mesa/default.nix

```
@@ -31,7 +31,7 @@ with stdenv.lib;
 let
   # Release calendar: https://www.mesa3d.org/release-calendar.html
   # Release frequency: https://www.mesa3d.org/releasing.html#schedule
-  version = "20.0.8"; # Update only to the final (last planned) release (i.e. X.Y.MAX)?
+  version = "20.1.3"; # Update only to the final (last planned) release (i.e. X.Y.MAX)?
```

```suggestion
  version = "20.1.3";
```

I thought this **might** make sense but I do now think it's actually a bad idea (since #92807 is already the first reported bug that is fixed with the next feature release and feature releases will most likely often include fixes that won't get backported, especially to an EOL release).